### ISSUE TITLE
Android 9

### ISSUE THREAD

**Issue by mimmi20** [browscap/browscap#1896]
Mozilla/5.0 (Linux; Android 9; Pixel Build/PPP3.180510.008) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.81 Mobile Safari/537.36

see also ua-parser/uap-core#324.

**Comment by jaydiablo** [comment]
This is what we've seen with that format so far this month:

```

Mozilla/5.0 (Linux; Android 9; Pixel 2 XL Build/PPP3.180510.008) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.136 Mobile Safari/537.36
--
Mozilla/5.0 (Linux; Android 9; Pixel 2 Build/PPP3.180510.008) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.81 Mobile Safari/537.36
Mozilla/5.0 (Linux; Android 9; Pixel XL Build/PPP3.180510.008) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.81 Mobile Safari/537.36
Mozilla/5.0 (Linux; Android 9; Pixel Build/PPP3.180510.008) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.81 Mobile Safari/537.36
Mozilla/5.0 (Linux; Android 9; Pixel 2 XL Build/PPP3.180510.008) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.81 Mobile Safari/537.36
Mozilla/5.0 (Linux; Android 9; Pixel 2 XL Build/PPP3.180510.008) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.68 Mobile Safari/537.36
Mozilla/5.0 (Linux; Android 9; PH-1 Build/PPR1.180510.022) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/67.0.3396.68 Mobile Safari/537.36
```

**Comment by mimmi20** [comment]
@jaydiablo I found these useragents in my collection:
`Mozilla/5.0 (Linux; Android 9.0; SP-9600B Build/KVT49L) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/30.0.0.0 Mobile Safari/537.36` and
`Mozilla/5.0 (Linux; U; Android 9.0; de-de; TP91A Build/JDQ39) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Safari/534.30`. But I cant find any device information.

Shall I add them as `general mobile`?

**Comment by jaydiablo** [comment]
Perhaps this is the TP91A: http://hitech-tabletpc.com/1-3-9-inch-tablet-pc.html/120338/  I feel like it's a spoof though, as that page shows that it's Android 4. I think it's unlikely that there would be off-brand Chinese tablets running Android 9.0 out there already.

The other one turns up here: https://www.handsetdetection.com/device-detection-database/platforms/android/9-0/ with a lot of other UAs that have `9.0` in the UA as the Android version on devices that I don't think support Android 9.0 (I suspect almost all of these UAs are spoofed, so this one probably is too).

It **seems** that Android 9 isn't going to have `9.0` as the Android version, but just `9`, which is what I searched for in our logs.  The devices in the UAs I found seem legit, as Google typically releases the new Android versions on their Pixel devices first (I don't know what `PH-1` is).  I'd be suspect of anything with `9.0` or `9.0.1` in the UA, at least right now.
